scripts/test/run-tests.sh

```diff
@@ -161,6 +161,20 @@ if [ "$TEST_ALL" == "1" -o "$TEST_PLATFORM" == "raspberrypi4_bookworm_64bit" ];
                    || test_result=$?
 fi
 
+if [ "$TEST_ALL" == "1" -o "$TEST_PLATFORM" == "raspberrypi5_bookworm_64bit" ]; then
+  MATCHED_A_TEST=1
+  # For this test we test compressed image to verify xz compression
```

Comment:
:ok_hand: But consider testing uncompressed - it saves a bit of time and we already have other tests covering xz compression.